### PR TITLE
Muzima 268- Apply validation against bounded to mUzima forms

### PR DESCRIPTION
**Pull request by ManishaRana1195**

### CODE REVIEW

File: Chw.html [hunk 1]
```diff
@@ -505,7 +505,7 @@ <h2 class="text-center">CDM CHW Form v 0.01</h2>
 <div class="section">
     <div class="form-group">
         <label for="encounter.provider_id_select">Provider Name:</label>
-        <input class="form-control" id="encounter.provider_id_select" type="text"
+        <input class="form-control validProviderOnly" id="encounter.provider_id_select" type="text"
```
Comment (nribeka) [inline Chw.html]: Can we replace the class validProviderOnly? I think the convention for class name is usually using dash. So my preference would be valid-provider-only.

File: Dysplasia.html [hunk 2]
```diff
@@ -1512,6 +1481,26 @@ <h2 class="text-center">Dysplasia Form v 0.01</h2>
         toggleValidationMessages(validateAlone($(this), colposcopyresultsValues, colposcopyresultsMessage));
     });
 
+    /* Start - Checking that the user entered provider exists in the list of possible providers */
```
Comment (nribeka) [inline Dysplasia.html]: Should this go to the core muzima javascript as the implementation seems to be similar in all instances?

File: Chw.html (hunk 1 above)
Comment (nribeka) [inline Chw.html]: I'm guilty of not doing this early in the development of the form, but I think moving forward we should do this :)

Comment (ManishaRana1195) [inline Chw.html]: We have made the above changes and have amended the original commit. Please merge the pull request.

**Pull request merged**